packages/docs-website/docs/docs/protocol-tutorial/outcomes.md

```diff
@@ -61,7 +65,7 @@ expect(decodeOutcome(encodedOutcome)).toEqual(outcome);
 
 ## Outcomes that guarantee
 
-Guarantee Asset Outcomes are similar to Allocation Asset Outcomes, only they not have any amounts. Their purpose is to simply express an ordering of destinations for a given asset holder (say, a given token).
+Guarantee Asset Outcomes are similar to Allocation Asset Outcomes, only they not have any amounts. Their purpose is to simply express an ordering of destinations for a given asset (say, a given token).
```

Comment:
```suggestion
Guarantee Asset Outcomes are similar to Allocation Asset Outcomes, only they do not have any amounts. Their purpose is to simply express an ordering of destinations for a given asset (say, a given token).
```

this word will have the great joy of living for a little while in this side-trunk.